### ISSUE TITLE
How can I support custom operators?

### ISSUE THREAD

**Issue by xunleiwu** [facebookresearch/fvcore#83]

**Comment by xunleiwu** [comment]
I have a model and `pprint.pprint(flops.unsupported_ops())` says that even `aten::mul` is not supported. So I tried this
`def fvcore_mul_flop_jit(
    inputs: typing.List[Any], outputs: typing.List[Any]
) -> typing.Counter[str]:
    """
    flops of aten::mul
    """
    
    flop_dict = Counter()
    flop_dict["mul"] = np.prod(inputs[0].shape)
    return flop_dict

custom_ops: Dict[str, Handle] = {"aten::mul": fvcore_mul_flop_jit}
flops = FlopCountAnalysis(model, (input,)).set_op_handle(**custom_ops)`

Running the program results this error:
`    flop_dict["mul"] = np.prod(inputs[0].shape)
AttributeError: 'torch._C.Value' object has no attribute 'shape'`

I have not found an example in the repo deriving flops from input/output tensor shapes. Would you please post some examples?  [These examples](https://github.com/facebookresearch/fvcore/blob/e05980b721dff2e0a9a21040b4cc51857b24cffb/tests/test_flop_count.py) are not helpful.
Thank you.

**Comment by ppwwyyxx** [comment]
https://github.com/facebookresearch/fvcore/blob/e5a3815eb1ae4562b2f47605e6ca43e8e6e87585/fvcore/nn/jit_handles.py#L18-L31

**Comment by xunleiwu** [comment]
> https://github.com/facebookresearch/fvcore/blob/e5a3815eb1ae4562b2f47605e6ca43e8e6e87585/fvcore/nn/jit_handles.py#L18-L31

@ppwwyyxx, Thank you very much.

"I have a model and pprint.pprint(flops.unsupported_ops()) says that even aten::mul is not supported." fvcore also reports aten::add, aten::div, aten::sub, aten::add_, aten::softmax, ... are not supported. Is it expected?

**Comment by ppwwyyxx** [comment]
Yes, it is expected because these small operators are often ignored.
